src/ActionExecutor/UserConfirmation.php

```diff
@@ -50,6 +51,27 @@ public function init()
         parent::init();
         $this->observeChanges();
         $this->addClass($this->size);
+    }
+
+    /**
+     * Properly set element id for this modal.
+     *
+     * @param Generic $action
+     *
+     * @throws \atk4\core\Exception
+     */
+    public function afterActionInit(Generic $action)
+    {
+        $getTableName = function ($arr) {
+            foreach ($arr as $k => $v) {
+                return is_numeric($k) ? $v : $k;
+            }
+        };
+
+        $table_name = is_array($action->getModel()->table) ? $getTableName($action->getModel()->table) : $action->getModel()->table;
```

Comment:
How model->table can be of type array ?

Reply:
@DarkSide666 - I got this from property param type
https://github.com/atk4/data/blob/d2d83c2e82ce23bdb8c61d99b78db177ee0da6f0/src/Model.php#L131

Is it still the case? If array is not supported then I can change it.

Reply:
I think it never is used as array and what's written in param description wasn't actually developed. But OK, let's make sure we also support array here for possible future use.